src/odemis/driver/actuator.py

```diff
@@ -666,6 +667,41 @@ def _get_rot_matrix(self, invert=False):
             rotation *= -1
         return RigidTransform(rotation=rotation).matrix
 
+    def _getAxesRange(self):
+        """
+        Calculate the axes range for position VA based on the conversion matrix
+        from dep's positon.
+        """
+        # Dep's range
+        range_y = self.axes["y"].range
+        range_x = self.axes["x"].range
+
+        # Calculate rectangular border points on the given range
+        min_x = range_x[0]
+        max_x = range_x[1]
+        min_y = range_y[0]
+        max_y = range_y[1]
+        coordinates = [[min_x, min_y],
+                       [min_x, max_y],
+                       [max_x, min_y],
+                       [max_x, max_y]]
+
+        # Calculate range based on conversion matrix
+        new_coordinates = []
+        for val in coordinates:
+            vpos = self._convertPosFromdep(val)
+            new_coordinates.append(vpos)
+
+        # Update default axes range and increase by buffer
+        # to account for movements at the extreme location
+        new_range = numpy.array(new_coordinates, dtype=float)
+        buff = 2.e-03
+        self.axes["x"].range = (min(new_range[:, 0] - buff),
+                                max(new_range[:, 0]) + buff)
+        self.axes["y"].range = (min(new_range[:, 1] - buff),
+                                max(new_range[:, 1] + buff))
```

Comment:
Could you please have all the `min/max(...) +/- buff` written the same? You currently have one with different parenthesis. I think you can always first compute the max/min and then do the buffer.